variable.go

```diff
@@ -485,17 +507,34 @@ func (vr *variableResolver) resolve(ctx *ExecutionContext) (*Value, error) {
 					if pv.IsNil() {
 						// Workaround to present an interface nil as reflect.Value
 						var empty any = nil
-						parameters = append(parameters, reflect.ValueOf(&empty).Elem())
+						val = reflect.ValueOf(&empty).Elem()
 					} else {
-						parameters = append(parameters, reflect.ValueOf(pv.Interface()))
+						val = reflect.ValueOf(pv.Interface())
 					}
 				} else {
 					// Function's argument is a *pongo2.Value
-					parameters = append(parameters, reflect.ValueOf(pv))
+					val = reflect.ValueOf(pv)
+				}
+
+				if val.Kind() == reflect.Invalid {
+					return nil, fmt.Errorf("calling a function using an invalid parameter")
 				}
+
+				if pv.IsKwarg() {
+					kwargs[pv.Name()] = pv
+				} else if len(kwargs) == 0 {
+					args = append(args, val)
+				} else {
+					return nil, fmt.Errorf("calling a function using a positional argument: %v, after a keyword argument", pv)
```

Comment:
can we give name of func also here. and other places.

Reply:
Name of function is included in the error message, `[Error (where: execution) in <string> | Line 1 Col 4 near 'testFunc']`